src/test/sighash_tests.cpp

```diff
@@ -125,10 +125,11 @@ BOOST_AUTO_TEST_CASE(sighash_test)
     std::cout << "[\n";
     std::cout << "\t[\"raw_transaction, script, input_index, hashType, signature_hash (result)\"],\n";
     #endif
-    int nRandomTests = 50000;
 
     #if defined(PRINT_SIGHASH_JSON)
-    nRandomTests = 500;
+    int nRandomTests = 500;
+    #else
+    int nRandomTests = 50000;
     #endif
     for (int i=0; i<nRandomTests; i++) {
```

Comment:
`nRandomTests` is undeclared in case of `!defined(PRINT_SIGHASH_JSON)`. There are too many `#if defined(PRINT_SIGHASH_JSON)` anyway...